webpack.prod.config.js

```diff
@@ -34,7 +34,7 @@ config.plugins = [
   // Stage also uses this config and dev uses this config when running npm run build
   new CopyPlugin([
     {
-      from: path.resolve(__dirname, 'public/REV1074'),
+      from: path.resolve(__dirname, 'public/static-checkout'),
```

Comment:
this is what pulls in the file into the dist folder (necessary to access it on prod)